src/client/client.d

```diff
@@ -312,8 +312,16 @@ Socket createSocket(string socketFile, ushort port)
 	}
 	else
 	{
-		socket = new Socket(AddressFamily.UNIX, SocketType.STREAM);
-		socket.connect(new UnixAddress(socketFile));
+		version(Windows)
+		{
+			fatal("Cannot use UNIX domain sockets on Windows.");
+			assert(0);
```

Comment:
Will do when Im home from work this evening. Don't think return 1 works here though, the function returns a Socket iirc.